scripts/provisioning-tests

```diff
@@ -183,6 +210,7 @@ echo "Waiting up to 5 minutes for rancher-provisioning-capi deployment"
   --exit-command "dump_rancher_logs" `# Dump logs to find out why webhook did not start` \
   "kubectl --kubeconfig /etc/rancher/k3s/k3s.yaml rollout status -n cattle-provisioning-capi-system deploy/capi-controller-manager &>/dev/null"
 
+( push_cattle_agent_image & )
```

Comment:
If you want https://github.com/rancher/kontainer-driver-metadata/blob/dev-v2.10/scripts/fetch-provisioning-tests to still work, I would recommend gating this with an environment variable like `PUSH_AGENT_IMAGE`, otherwise the KDM provisioning tests will break.